### PR TITLE
Add introduction to README

### PR DESCRIPTION
**Pull request by domcleal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
-      server_foreman        => false,
-      server_reports        => 'store',
-      server_external_nodes => '',
-    }
+## PuppetDB integration with puppetlabs/puppetdb
+
+The Puppet master can be configured to export catalogs and reports to a
+PuppetDB instance, using the puppetlabs/puppetdb module.  Use its
+`puppetdb::server` class to install PuppetDB and this module to configure the
+master.
```
Comment (ekohl) [inline README.md]: Puppet master?

**Comment by ekohl** [comment]
Looks like a long overdue update. :+1:

**Comment by mmoll** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (domcleal) [inline README.md]: Fixed, plus a couple of other poor pieces of English.

**Comment by mmoll** [comment]
thanks @domcleal!

**Pull request merged**